content/virtualization.md

```diff
@@ -0,0 +1,199 @@
+
+##### Windows Driver Downloads
+
+- Windows Guest Tools: [https://www.spice-space.org/download/windows/spice-guest-tools/spice-guest-tools-latest.exe](https://www.spice-space.org/download/windows/spice-guest-tools/spice-guest-tools-latest.exe)
+- Spice WebDAVd: [https://www.spice-space.org/download/windows/spice-webdavd/spice-webdavd-x64-latest.msi](https://www.spice-space.org/download/windows/spice-webdavd/spice-webdavd-x64-latest.msi)
+
+> While trying to download the Spice WebDAVd file in the Edge browser, if I clicked the link, I was taken to a page trying to display the file as text instead of downloading the file. This ends up showing a lot of odd letters and characters.
+>
+> When trying the download a few more times, I either saw same window with random text, or I received a "This page isn't responding" prompt.
+>
+> If either of these happen when you download, right-click the link above and select "Save Link As..." and save the file to your Downloads folder.
+
+## Virtualiaztion Features
+
+Now that the VM is set up, and drivers are installed, we can now use the advanced features of GNOME Boxes.
+
+Most of these features can be found in the VM's settings menu. To reach the VM settings menu, click on your VM in the GNOME Boxes list to start the VM. You can now select the <kbd>&vellip;</kbd> button at the top right of GNOME Boxes, then select "Properties" to open the VM settings menu.
+
+### Copy and Paste
+
+Once your VM has the `spice-vdagent` or an equivelent package installed on your VM, reboot the VM once to load this driver. At this point, you will now be able to copy and paste between your VM and the Host just like copy and pasting between applications.
+
+### Share a USB Device
+
+![Devices and Shares](/images/gnome-boxes/boxes-vmprefs-devices_and_shares.png)
+
+One of the features of GNOME Boxes, is the ability to transfer a USB device connected to your Host to the VM. You can find this option in the VM settings menu, under the "Devices & Shares" tab. On this tab, under the "USB devices" heading, you will see a list of the USB devices found on your system. If you click the toggle bottom on the right, the USB device will disappear from your Host operating system, and it will appear inside your VM.
+
+### Share a Folder
+
+_For this guide, we used GNOME-based Linux distributions, so the VMs UI closely matched Pop!_OS and Ubuntu. Accessing shared folders with other desktop environments is possible, but you will need to refer to their documentation for the details_
```

Comment:
Minor formatting - this renders odd for me, not sure if this is just on my end or if this is a markdown syntax issue 
![image](https://user-images.githubusercontent.com/80931038/159072246-73bca338-8113-44ad-ada9-9ee213b420e4.png)

Reply:
I corrected this in my latest commit. :)